roles/deploy_user/files/id_rsa

```diff
@@ -0,0 +1,136 @@
+$ANSIBLE_VAULT;1.1;AES256
+30343436363139383434343533623335353336666463363138363462653966636234303130633330
```

Comment:
Don't we have some automated system that prevents us from committing unencrypted vault files? I can't find it. And if that is the case, would that system know to apply that check to this file?

Reply:
Good catch. There's a precommit hook in `hooks/pre-commit`; instructions for installing it are in the readme, but right now it only looks for vault files by name. It would be good to add this one to the list.